packages/core/content-manager/admin/src/history/components/VersionContent.tsx

```diff
@@ -2,41 +2,75 @@ import * as React from 'react';
 
 import { Form, useField, useStrapiApp } from '@strapi/admin/strapi-admin';
 import {
+  Alert,
   Box,
   ContentLayout,
   FieldLabel,
   Flex,
   Grid,
   GridItem,
+  Link,
+  Tooltip,
   Typography,
 } from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
 
+import { COLLECTION_TYPES } from '../../constants/collections';
+import { DocumentStatus } from '../../pages/EditView/components/DocumentStatus';
 import {
   InputRenderer,
   type InputRendererProps,
 } from '../../pages/EditView/components/InputRenderer';
+import { getRelationLabel } from '../../utils/relations';
 import { useHistoryContext } from '../pages/History';
 
 import type { RelationsFieldProps } from '../../pages/EditView/components/FormInputs/Relations';
+import type { RelationResult } from '../../services/relations';
+
+const StyledAlert = styled(Alert)`
+  button {
+    display: none;
+  }
+`;
 
 /* -------------------------------------------------------------------------------------------------
  * CustomRelationInput
  * -----------------------------------------------------------------------------------------------*/
 
+const LinkEllipsis = styled(Link)`
+  display: block;
+
+  & > span {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: block;
+  }
+`;
+
 const CustomRelationInput = (props: RelationsFieldProps) => {
-  const field = useField(props.name);
+  const { formatMessage } = useIntl();
+  const {
+    value: { results, meta },
+  } = useField(props.name);
 
   return (
     <Box>
       <FieldLabel>{props.label}</FieldLabel>
-      {field.value.results.length === 0 ? (
+      {results.length === 0 ? (
         <Typography>No content</Typography>
       ) : (
```

Comment:
I'm waiting on @lucasboilly for the empty state design, this is a placeholder

Reply:
Done, please find it [here](https://www.figma.com/file/XYJUrJn9i9ius3bOUSL4fn/Content-History?type=design&node-id=1444%3A22532&mode=design&t=RPZeMPo0TUyWrB1I-1) :)